Formula/geesefs.rb

```diff
@@ -0,0 +1,24 @@
+class Geesefs < Formula
+  desc "Finally, a good FUSE FS implementation over S3"
```

Comment:
```suggestion
  desc "FUSE FS implementation over S3"
```

Reply:
The idea is that there are other fuse-s3 implementations, for example s3fs, and also goofys which geesefs is a fork of, but geesefs is much faster and has more features :-) that phrase is actually from geesefs github page. I can change it to, say, "high performance fuse s3" or "fuse s3 with parallelism" or something like that